speechbrain/lobes/models/huggingface_transformers/gpt.py

```diff
@@ -0,0 +1,151 @@
+"""This lobe enables the integration of huggingface pretrained GPT2LMHeadModel model.
+
+Transformer from HuggingFace needs to be installed:
+https://huggingface.co/transformers/installation.html
+
+Authors
+ * Pooneh Mousavi 2023
+ * Simone Alghisi 2023
+"""
+
+import logging
+from torch import Tensor
+import torch
+
+from speechbrain.lobes.models.huggingface_transformers.huggingface import (
+    HFTransformersInterface,
+)
+
+
+logger = logging.getLogger(__name__)
+
+
+class GPT(HFTransformersInterface):
```

Comment:
@poonehmousavi : This class defines the GPT model. Could you please complete the docstring? I think it lacks quite a lot of the parameters used by the class. I could do it but I think you're more familiar with the class.